static/webxdc-preload.js

```diff
@@ -9,19 +9,37 @@
   let callback = null
   var last_serial = 0
   let setUpdateListenerPromise = null
+  let is_running = false
+  let scheduled = false
```

Comment:
Why these names are in snake case?

Reply:
no particular reason, there are no real/clear rules for casing in desktop, I guess camel case is preferred generally, but if you look above `last_serial` is also snake case.